source/localizable/smart_cart/_order_object.html.md.erb

```diff
@@ -94,3 +107,27 @@ Name                                                       | Type    | Descripti
 `line_item_rejection_reason.id`                            | Integer | The rejection reason ID to use
 `line_item_rejection_reason.label`                         | String  | A human readable label
 `line_item_rejection_reason.requires_available_quantity`   | Boolean | Whether or not `available_quantity` is required
+
+#### Invoice details
+
+This attribute is present only when a user requests an invoice for the order. A user
+may also request vat exclusion for several line items.
+
+
+Name                           | Type    | Description
+:----------------------------: | :-----: | :----------------------------------------------:
+`company`                      | String  | Company name
+`profession`                   | String  | Company profession
+`vat_number`                   | String  | Company's vat number
+`doy`                          | String  | Company's public economic service
+`address`                      | String  | Billing address
+`vat_exclusion_requested`      | Boolean | Vat exclusion requested under 39A
```

Comment:
I suggest we write it like this:
`VAT exclusion requested (Υπηρεσία άρθρου 39α)`, με το `Υπηρεσία άρθρου 39α` link στο https://www.aade.gr/epiheiriseis/forologikes-ypiresies/fpa/yperesia-arthroy-39a

(39α is greek-only)